src/Illuminate/Support/helpers.php

```diff
@@ -883,3 +883,15 @@ function with($object)
         return $object;
     }
 }
+
+if (! function_exists('is_winnt')) {
+    /**
+     * Determine whether the current envrionment is Windows NT based.
+     *
+     * @return bool
+     */
+    function is_winnt()
+    {
+        return substr(strtoupper(php_uname('s')), 0, 3) === 'WIN';
```

Comment:
Indeed, but in practice it works perfectly and will do for a very long time.

In fact, the probabilty that someone releases some OS named "WINner Linux" is higher than seeing another OS with backslash separators.

Even Symfony uses this method, if this could reassure you.